sbin/prepareWorkspace.sh

```diff
@@ -164,6 +166,27 @@ checkingAndDownloadingAlsa()
   fi
 }
 
+checkFingerPrint() {
+  local sigFile="$1"
+  local fileName="$2"
+  local publicKey="$3"
+  local expectedFingerPrint="$4"
+
+  rm /tmp/public_key.gpg || true
+  gpg --output /tmp/public_key.gpg --dearmor "${SCRIPT_DIR}/sig_check/${publicKey}.asc"
```

Comment:
Do we know gpg is installed on all hosts?

Reply:
Does `${SCRIPT_DIR}/sig_check/` always exist?  Hmm, I think it does because you're adding .asc files into it below